dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1112,6 +1112,150 @@ def median_abs_deviation(self):
                                    [edge_prev, edge_cur])
         return median_abs_dev
 
+    @property
+    def median_abs_deviation(self):
+        """
+        Get median absolute deviation.
+
+        :return: median absolute deviation
+        """
+        if not self._has_histogram:
+            return np.nan
+
+        bin_counts = self._stored_histogram['histogram']['bin_counts']
+        bin_edges = self._stored_histogram['histogram']['bin_edges']
+
+        # normalize bin counts
+        bin_counts = bin_counts.astype(float)
+        normalized_bin_counts = bin_counts / np.sum(bin_counts)
+
+        # calculate deviations from median
+        median = self.quantiles[499]
+        bin_edges = bin_edges - median
+
+        # find the break point to fold the deviation list
+        id_zero = None
+        for i in range(len(bin_edges)):
+            if bin_edges[i] > 0:
+                id_zero = i
+                break
+
+        # if all bin edges are positive or negative (no break point),
+        # the median value is actually 0
+        if id_zero is None or id_zero == 1:
```

Comment:
Should the negative break point be `1` or `len(bin_edges) - 1`?

Reply:
I don't get it. Basically, id_zero is None with all edges are negative and 1 when all edges are positive

Reply:
i above is equal to  a value within: `range(len(bin_edges))` which are integers between `0` and the `len(bin_edges) - 1`

Reply:
I see so you are saying if all are positive, it is already > 0 at the first bin. but `None` bc the iteration no positive # existed.